devel/oasdiff/Portfile

```
@@ -19,10 +20,33 @@ license             Apache-2
 maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 
-checksums           rmd160  6af584b7721862d33f3d6ada056abb2954be7527 \
-                    sha256  9e000211f5a9561ffeb6ab078e02177349f1f262a69f0619d9b0a507cc73498e \
-                    size    357982
+checksums           ${distname}${extract.suffix} \
+                    rmd160  c02fd8a6f4b145a8c811f4a247164c83b8cc11c8 \
+                    sha256  bd06a38e62657634ab95ebc06174580fd3840fc07d8e5646eaed229d6dff424f \
+                    size    404414
+
+post-build {
+    system "${worksrcpath}/${name} completion bash > ${worksrcpath}/${name}.bash"
```

Just a tip, you can use `-W <dir>` to set a working directory for the `system` command.

```suggestion
    system -W ${worksrcpath} "${name} completion bash > ${name}.bash"
```

Not saying this should be changed, but just an FYI.